src/ser/mod.rs

```diff
@@ -207,6 +208,7 @@ impl Default for PrettyConfig {
             indentor: default_indentor(),
             separate_tuple_members: default_separate_tuple_members(),
             enumerate_arrays: default_enumerate_arrays(),
+            extensions: default_extensions(),
```

Comment:
should probably just use `Extensions::default()` here

Reply:
ah, good point. will fix that right up.